yellowbrick/utils/helpers.py

```diff
@@ -28,6 +28,29 @@
 
 
 ##########################################################################
+## Character Mapping helper function
+##########################################################################
+
+def character_mapping(array, mapping):
+    """
+
+    Parameters
+    ----------
+    array: iterable
+        an iterable object containing predictions from a classifier
+
+    Returns
+    -------
+    array : list
+        a list of predicted values after applying a given mapping
+
+    """
+    array = [str(x) for x in array]
+    for key, value in mapping.items():
+        array = [x.replace(value, key) for x in array]
```

Comment:
Ah, so `str.replace` -- replaces every instance if a character with the value from the key. E.g. `"bcb".replace("b", "a")` results in the string `"aca"`. Given the use case for target/class mapping, may I suggest:

```python
array = [mapping[x] for x in array] 
```

Also I'd note that the input is likely to be a `np.array`. I investigated `np` replace methods, but most of them use masks, e.g. `array[array == x] = mapping[x]` will replace all values of x with the mapping character, but this is probably slower than looping through every element in the array since you'll be replacing all values anyway.